buster/docparser.py

```diff
@@ -148,23 +148,24 @@ def generate_embeddings_parser(root_dir: str, output_filepath: str, source: str)
     return generate_embeddings(documents, output_filepath)
 
 
-def documents_to_db(documents: pd.DataFrame, output_filepath: str):
+def documents_to_db(
+    documents: pd.DataFrame,
+    documents_manager: DocumentsManager,
+):
     logger.info("Preparing database...")
-    documents_manager = get_documents_manager_from_extension(output_filepath)(output_filepath)
     sources = documents["source"].unique()
     for source in sources:
         documents_manager.add(source, documents)
-    logger.info(f"Documents saved to: {output_filepath}")
+    logger.info(f"Documents saved to documents manager")
```

Comment:
should each documents manager have a `__name__` property so we can differentiate them in the logs?

Reply:
good point, although it's `__repr__` that must be defined for this.